### PR TITLE
[docs, dagster-databricks] add docs for using Pipes with existing clusters

### PR DESCRIPTION
**Pull request by danielgafni**
## Changelog

Insert changelog entry or "NOCHANGELOG" here.

- [x] `DOCS` _(added docs for using Databricks Pipes with existing clusters)_

### CODE REVIEW

**Comment by danielgafni** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24684?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#24684** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24684?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#24526** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24526?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @danielgafni and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/next/next-env.d.ts [hunk 1]
```diff
@@ -2,4 +2,4 @@
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
-// see https://nextjs.org/docs/pages/building-your-application/configuring/typescript for more information.
+// see https://nextjs.org/docs/basic-features/typescript for more information.
```
Comment (danielgafni) [inline docs/next/next-env.d.ts:5]: no idea where is this coming from

File: docs/content/concepts/dagster-pipes/databricks.mdx [hunk 2]
```diff
@@ -120,9 +124,10 @@ Let's review what's happening in this code:
...
+    - If `new_cluster` is set, then also setting `new_cluster.cluster_log_conf.dbfs` will allow the <PyObject object="PipesDatabricksClient" module="dagster_databricks" /> will automatically set up <PyObject object="PipesDbfsLogReader" module="dagster_databricks" /> objects for `stdout` and `stderr` of the driver node. These will periodically forward the `stdout` and `stderr` logs written by Databricks back to Dagster. **Note**: Because Databricks only updates these log files every five minutes, that is the maximum frequency at which Dagster can forward the logs.
+    - If `existing_cluster_id` is set, <PyObject object="PipesDatabricksClient" module="dagster_databricks" /> won't be able to forward `stdout` and `stderr` driver logs to Dagster. Also, this mode requires passing <PyObject object="PipesCliArgsParamsLoader" module="dagster_pipes" /> to <PyObject object="open_dagster_pipes" module="dagster_pipes" /> in the Python script that is executed on Databricks. This is because setting environment variables is only possible when creating a new cluster, so we have to resort to passing Pipes parameters as command-line arguments.
```
[the remark this answers is not in the document]
Comment (smackesey) [reply docs/content/concepts/dagster-pipes/databricks.mdx:130]: also "requires passing" -> "requires passing an instance of"

**Review by smackesey — CHANGES_REQUESTED** [review]

File: docs/content/concepts/dagster-pipes/databricks.mdx [hunk 3]
```diff
@@ -214,7 +215,7 @@ Before we go any futher, let's review what this script does:
 
   Both objects write temporary files on DBFS for communication between the orchestration and external process. The <PyObject object="PipesDbfsContextLoader" module="dagster_pipes" /> and <PyObject object="PipesDbfsMessageWriter" module="dagster_pipes" /> match a corresponding `PipesDbfsContextInjector` and `PipesDbfsMessageReader` on the orchestration end, which are instantiated inside the <PyObject object="PipesDatabricksClient" module="dagster_databricks" />.
 
-- **Passes the context loader and message writer to the <PyObject object="open_dagster_pipes" module="dagster_pipes" /> context manager**, which yields an instance of <PyObject object="PipesContext" module="dagster_pipes" /> called `pipes`.
+- **Passes the context loader and message writer to the <PyObject object="open_dagster_pipes" module="dagster_pipes" /> context manager**, which yields an instance of <PyObject object="PipesContext" module="dagster_pipes" /> called `pipes`. **Note**: when using `exiting_cluster_id`, you must also import `PipesCliArgsParamsLoader` and pass an instance of it to `open_dagster_pipes` as the `params_loader` parameter.
```
Comment (smackesey) [inline docs/content/concepts/dagster-pipes/databricks.mdx:218]: "exiting" -> "existing"

**Review by smackesey — APPROVED** [review]
one typo to fix but otherwise lgtm

**Pull request merged**